build fails on ubuntu 14.04.1 "TrustyThar"

also on the [buildbot](https://buildbot.openlighting.org/builders/buildcheck-ola-ubuntu-tahr-i686-noopenslp/builds/1581/steps/make/logs/stdio)
build log:
```bash
/bin/bash ./libtool  --tag=CXX   --mode=compile g++ -DHAVE_CONFIG_H -I.    -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -pthread -Werror -I/usr/include/libusb-1.0   -g -O2 -pthread -MT plugins/usbdmx/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.lo -MD -MP -MF plugins/usbdmx/.deps/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.Tpo -c -o plugins/usbdmx/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.lo `test -f 'plugins/usbdmx/AsyncPluginImpl.cpp' || echo './'`plugins/usbdmx/AsyncPluginImpl.cpp
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -pthread -Werror -I/usr/include/libusb-1.0 -g -O2 -pthread -MT plugins/usbdmx/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.lo -MD -MP -MF plugins/usbdmx/.deps/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.Tpo -c plugins/usbdmx/AsyncPluginImpl.cpp  -fPIC -DPIC -o plugins/usbdmx/.libs/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.o
plugins/usbdmx/AsyncPluginImpl.cpp: In member function 'virtual bool ola::plugin::usbdmx::AsyncPluginImpl::Start()':
plugins/usbdmx/AsyncPluginImpl.cpp:98:28: error: expected type-specifier before 'LibUsbHotplugThread'
     m_usb_thread.reset(new LibUsbHotplugThread(
                            ^
make[2]: *** [plugins/usbdmx/plugins_usbdmx_libolausbdmx_la-AsyncPluginImpl.lo] Error 1
```

--- a/plugins/usbdmx/LibUsbThread.cpp
+++ b/plugins/usbdmx/LibUsbThread.cpp
@@ -60,7 +60,7 @@ void LibUsbThread::JoinThread() {
 // LibUsbHotplugThread
 // -----------------------------------------------------------------------------
 
-#if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000102)
+#if HAVE_LIBUSB_HOTPLUG_API
 LibUsbHotplugThread::LibUsbHotplugThread(libusb_context *context,
                                          libusb_hotplug_callback_fn callback_fn,
                                          void *user_data)

--- a/plugins/usbdmx/LibUsbThread.h
+++ b/plugins/usbdmx/LibUsbThread.h
@@ -23,6 +23,10 @@
 
 #include <libusb.h>
 
+#if HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #include "ola/base/Macro.h"
 #include "ola/thread/Thread.h"
 
@@ -128,7 +132,7 @@ class LibUsbThread : private ola::thread::Thread {
   ola::thread::Mutex m_term_mutex;
 };
 
-#if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000102)
+#if HAVE_LIBUSB_HOTPLUG_API
 
 /**
  * @brief The hotplug version of the LibUsbThread.

